files/lib/system/exporter/WBB4xExporter.class.php

```diff
@@ -4277,6 +4277,9 @@ private function exportCategories($objectType, $importer, $offset, $limit)
             if (\str_starts_with($row['description'], 'wcf.category')) {
                 $i18nValues[] = $row['description'];
             }
+            if (\unserialize($row['additionalData']) !== false) {
+                $i18nValues[] = $row['additionalData'];
+            }
```

Comment:
This does not look correct. Neither `additionalData` nor `eventColor` are `i18nValues`.